app/app.go

```diff
@@ -21,9 +21,9 @@ import (
 	"github.com/neutron-org/neutron/docs"
 
 	"github.com/neutron-org/neutron/app/upgrades"
+	v030 "github.com/neutron-org/neutron/app/upgrades/v0.3.0"
```

Comment:
a minor suggestion: let's make this one and the rest as `v0_3_0`? otherwise it could be a little pain in the ass in the future when we have releases like 2.0.15 as `2015`. we'd better stick with a more readable pattern for versions

Reply:
Linter doesn't like this